### DISCUSSION TITLE
Hi! is Nuxt using the power of Astro?

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#23004] [Questions]
Currently, the Astro 3 has an update about View Transition which is great because it works in SSR https://astro.build/blog/astro-3/. If not I think this is a good feature for nuxt. Especially the warp sample https://astro-records.pages.dev/.

**Comment by Hebilicious** [comment]
Nuxt added view transitions support before Astro did https://nuxt.com/docs/getting-started/transitions#view-transitions-api-experimental

However I do think we could flesh them out more in Nuxt : 

- Write a dedicated section in the docs
- Add some usage convenience 
- Improve the showcase website: https://movies.nuxt.space/movie/615656

**Comment by dosstx** [comment]
I was looking to implement a similar idea the way Nuxt did it with the movies demo, however, I find I need a tutorial or something to get started. 

Perhaps someone can create a sandbox for how Nuxt did the transitions for the "movie posters to movie page" on their movie site? I'd like to do something similar.

**Comment by kalnode** [comment]
By "View Transition" is everyone talking about the future native HTML View Transitions?
https://developer.mozilla.org/en-US/docs/Web/API/View_Transition_API

**Reply by JenuelDev** [reply]
yup yup
